build/pause/Makefile

```diff
@@ -122,12 +122,12 @@ bin/wincat-windows-${ARCH}: windows/wincat/wincat.go
 
 container: .container-${OS}-$(ARCH)
 .container-linux-$(ARCH): bin/$(BIN)-$(OS)-$(ARCH)
-	docker buildx build --pull --output=type=${OUTPUT_TYPE} --platform ${OS}/$(ARCH) \
+	docker buildx build --provenance=false --sbom=false --pull --output=type=${OUTPUT_TYPE} --platform ${OS}/$(ARCH) \
```

Comment:
what version does this require?

Reply:
@pacoxu do you have the docker buildx version handy where these flags were introduced ? in older OSs with older version of docker , the builds are failing..